### PR TITLE
Update replace-license-key.adoc

### PR DESCRIPTION
**Pull request by steftux**
Windows procedure: add wsl hint and powershell command. Use variable for rtfctl apply command

# Writer's Quality Checklist

Before merging your PR, did you:

- [x ] Run spell checker
- [x ] Run link checker to check for broken xrefs
- [x ] Check for orphan files
- [x ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [x ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ x] If applicable, verify that the software actually got released

### CODE REVIEW

**Comment by steftux** [comment]
I just signed [salesforce-cla](https://github.com/apps/salesforce-cla) so cla:missing label should be removed

**Comment by IsaacEldridge** [comment]
Hey Stefan-- Thanks for the contribution. Can you double-check or try to sign the CLA again? If there isn't an issue on your end, I'll run this by our docs-gh infra team. Thanks!

**Review by kmendiratta-crm — APPROVED** [review]

File: modules/ROOT/pages/replace-license-key.adoc [hunk 1]
```diff
@@ -20,35 +20,33 @@ The `rtfctl` utility is required to replace your Mule license key on Runtime Fab
...
+BASE64_ENCODED_LICENSE=$(base64 -w0 license.lic)
 ----
-** On Windows, a shell terminal emulator (such as cygwin) or access to a Unix-based 
-computer is required.
-
-... Transfer to your Unix environment if necessary.
-... Run the following command to Base64 encode the license key:
+** On Windows you can either 
+*** use a WSL or Cygwin shell which include the base64 tool and use the above Unix command or 
+*** use the folloing Powershell command instead:
```
Comment (steftux) [inline modules/ROOT/pages/replace-license-key.adoc:32]: I meant to write "following"

**Comment by steftux** [comment]
> Hey Stefan-- Thanks for the contribution. Can you double-check or try to sign the CLA again? If there isn't an issue on your end, I'll run this by our docs-gh infra team. Thanks!

Hi Isaac, when I press on the button I come to a page saying "You already signed the CLA on 2022-05-02"

**Comment by steftux** [comment]
Hi @IsaacEldridge - as this PR can not be proceed (and I also added another option for windows users) I created a new PR which should be processed instead of this one. the pull request is  #690 Thank you

**Pull request closed without merging**